pkg/views/login/create.go

```diff
@@ -49,9 +53,18 @@ func CreateView(loginView *LoginView) {
 			huh.NewInput().
 				Title("Name of Credential").
 				Value(&loginView.Name).
+				Description("Name of credential to be stored in the harbor config file.").
+				PlaceholderFunc(func() string {
+					return fmt.Sprintf("%s@%s", loginView.Username, utils.SanitizeServerAddress(loginView.Server))
+				}, &loginView).
+				SuggestionsFunc(func() []string {
+					return []string{
+						fmt.Sprintf("%s@%s", loginView.Username, utils.SanitizeServerAddress(loginView.Server)),
+					}
+				}, &loginView).
 				Validate(func(str string) error {
 					if str == "" {
```

Comment:
Update `Name of Credential` `validate block` with this code 

```suggestion
				Validate(func(str string) error {
					if str == "" {
						loginView.Name = fmt.Sprintf("%s@%s", loginView.Username, utils.SanitizeServerAddress(loginView.Server))
						return nil
					}
					return nil
				}),

```

This ensures that when `Credential name` is empty it is autofilled with the default value.